marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/entities/search/EntitySearchManager.java

```diff
@@ -104,7 +104,7 @@ public StringHandle search(SearchQuery searchQuery) {
                 // We have some awkwardness here where the input is 'entityName', but as of 5.3.0, the "entityTypeIds"
                 // property is capturing entity names, which are expected to double as collection names as well
                 ServerTransform searchResultsTransform = new ServerTransform("hubEntitySearchTransform");
-                searchResultsTransform.put("entityName", entityTypeCollections[0]);
+                searchResultsTransform.put("entityNames", entityTypeCollections);
```

Comment:
I'd rather this still be "entityName", and then if the request has multiple entity names (which corresponds to "all entities", though in the future, this ideally will be captured via some other mechanism than the UI sending all entity names), then don't include this parameter. 

I think that keeps the search transform cleaner. Instead of throwing an error if entityName isn't specified, it'll say - Oh, there's no entityName selected, so the user is searching all entities. Thus, I'll return a generic set of properties instead of an entity-specific set of properties.

Reply:
Done.